array/DNAm/analysis/methodsDevelopment/compareNormalisation.r

```diff
@@ -4,56 +4,90 @@
 ##
 ## Purpose of script: compare effects of normalisation across all samples and within cell type.
 ##
-## Author: Eilis Hannon
-##
-## Date Created: 2020
-##
 ##---------------------------------------------------------------------#
 
 #----------------------------------------------------------------------#
 # NOTES
 #----------------------------------------------------------------------#
-# uses wateRmelon metrics to compre normalised datasets
+# uses wateRmelon metrics to compare normalisation strategies
 
 #----------------------------------------------------------------------#
 # DEFINE PARAMETERS
 #----------------------------------------------------------------------#
+set.seed(1234)
 args<-commandArgs(trailingOnly = TRUE)
 dataDir <- args[1]
+refPath <- args[2]
+
 gdsFile <-file.path(dataDir, "2_gds/raw.gds")
 normgdsFile<-sub("\\.gds", "Norm.gds", gdsFile)
+qcOutFolder<-file.path(dataDir, "/2_gds/QCmetrics")
 resPath<-file.path(dataDir, "4_analysis/methodsDevelopment/")
 
+cellTypes <- c("Double-", "NeuN+", "Sox10+")

```

Comment:
add to config file?

Reply:
This is specific to this analysis so I am going to leave here for now.